cmds/dump.c

```diff
@@ -1209,6 +1215,130 @@ static void print_flame_footer(struct uftrace_dump_ops *ops,
 	graph_remove_task();
 }
 
+static void print_dot_header(struct uftrace_dump_ops *ops,
+			     struct ftrace_file_handle *handle,
+			     struct opts *opts)
+{
+	pr_out("# version\":\"uftrace %s\",\n", UFTRACE_VERSION);
+
+	if (handle->hdr.info_mask & (1UL << CMDLINE))
+		pr_out("# command_line \"%s\"\n", handle->info.cmdline);
+
+	pr_out("digraph \"");
+	print_flame_header(ops, handle, opts);
+	pr_out("%s", (&handle->info)->exename);
+	pr_out("\" { \n");
+	pr_out("\n\t# Attributes \n");
+	pr_out("\tsplines=ortho;\n");
+	pr_out("\tconcentrate=true;\n");
+	pr_out("\tnode [shape=\"rect\",fontsize=\"7\",style=\"filled\"];\n");
+	pr_out("\tedge [fontsize=\"7\"];\n\n");
+}
+
+static void print_dot_task_start(struct uftrace_dump_ops *ops,
+				 struct ftrace_task_handle *task)
+{
+	print_flame_task_start(ops, task);
+}
+
+static void print_dot_inverted_time(struct uftrace_dump_ops *ops,
+				    struct ftrace_task_handle *task)
+{
+}
+
+static void print_dot_task_rstack(struct uftrace_dump_ops *ops,
+				  struct ftrace_task_handle *task, char *name)
+{
+	print_flame_task_rstack(ops, task, name);
```

Comment:
Please do not use flame graph data structure for other purpose.  Or make it to safe for sharing (I guess it's already so, but..)